content/concept-accounts.md

```diff
@@ -0,0 +1,123 @@
+# RCL Accounts
+
+An "Account" in the Ripple Consensus Ledger represents a holder of XRP and a sender of transactions. The core elements of an account are:
+
+- An identifying **address**, such as `rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn`
+- An account can send [transactions](reference-transaction-format.html) to modify the Ripple Consensus Ledger, for a variety of purposes including:
+    - Sending XRP to a recipient, immediately or through locking mechanisms like payment channels and escrow.
+    - Issuing, redeeming, or sending non-XRP assets and currencies.
+    - Trading in the decentralized exchange.
+    - Changing settings such as authorization settings, balance limits, or the behavior of assets issued by the account.
+- An **XRP balance**. Some of this XRP is set aside for the [Reserve](concept-reserves.html).
+- A **sequence number**, starting at 1 and increasing with each transaction sent from this account. No transaction can be included in a ledger unless the transaction's sequence number matches its sender's next sequence number.
+- A **history of transactions** that affected this account and its balances.
+
+In the ledger's data tree, an account's core data is stored in the [AccountRoot](reference-ledger-format.html#accountroot) ledger node type. An account can also be the owner (or partial owner) of several other types of data.
+
+**Tip:** An "Account" in the Ripple Consensus Ledger is somewhere between the financial usage (like "bank account") and the computing usage (like "UNIX account"). Non-XRP currencies and assets aren't stored in an RCL Account itself; each such asset is stored in an accounting relationship called a "Trust Line" that connects two parties.
+
+
+## Addresses
+
+{% include 'data_types/address.md' %}
+
+The conversion from a public key to an address involves a one-way hash function, so it is possible to confirm that a public key matches an address but it is impossible to derive the public key from the address alone. (This is part of the reason why signed transactions include the public key _and_ the address of the sender.) For more technical details, see [Address Encoding](#address-encoding).
+
+The process of "creating" an address is the purely mathematical task of generating a key pair and calculating the address from it, so an address can exist separately from its presence as an account object in the ledger. To actually make an account for an address, someone must send the address enough XRP to satisfy the reserve. This is called _funding_ the account. Whether or not the address is a funded account, you can use the address (and the key pair it's derived from) as a [regular key](reference-transaction-format.html#setregularkey) or as part of a [signer list](reference-transaction-format.html#multi-signing).
```

Comment:
That's a terrific revision. Much clearer and it breaks up the address/account relationship nicely. I didn't even break a sweat when I read "strictly."